src/views/tickets/Ticket.js

```diff
@@ -50,51 +52,60 @@ export default class Ticket extends Component {
       }
     } catch (error) {
       // todo: display the error on the page
```

Comment:
Can you take out this comment now that the error is being shown?

Reply:
Sure, I forget to do this.